### ISSUE TITLE
[FEATURE][docbot] Need function to clean up existing models after tests

### ISSUE THREAD

**Issue by dtaivpp** [opensearch-project/demos#53]
### Is your feature request related to a problem?
There needs to be a function that can delete all of the items that were used when testing from connectors to indexes. This is a sample of the order that is needed to delete all the artifacts created when using a model.  

```
POST /_plugins/_ml/models/<MODEL_ID>/_undeploy
DELETE /_plugins/_ml/models/<MODEL_ID>
DELETE /_plugins/_ml/connectors/<CONNECTOR_ID>
DELETE _ingest/pipeline/cohere-ingest-pipeline
DELETE cohere-index
```

**Comment by ortegaa32** [comment]
Could I be assigned to this issue please?

**Comment by dtaivpp** [comment]
Assigned!

**Comment by dtaivpp** [comment]
This section of code will have the model information we will need when undeploying. 

https://github.com/opensearch-project/demos/blob/9e93460a2dfd577d91777d5da9f306bae92504eb/demo/docbot/cluster_bootstrap.py#L13-L17

**Comment by dtaivpp** [comment]
Closed with #73
